.pullapprove.yml

```diff
@@ -1,8 +1,10 @@
 approve_by_comment: true
-approve_regex: ^LGTM
+approve_regex: '^(lgtm|LGTM)'
```

Comment:
On Tue, Jul 26, 2016 at 07:48:39PM -0700, Qiang Huang wrote:
> > @@ -1,8 +1,10 @@
> >  approve_by_comment: true
> > -approve_regex: ^LGTM
> > +approve_regex: '^(lgtm|LGTM)'
>
> I think we should just stick to `LGTM` as `lgtm` usually used as
> approve by someone who is not a maintainer.

You don't need to claim namespace for non-maintainers because they
aren't listed under the ‘reviewers’ section.